environment.yml

```diff
@@ -0,0 +1,30 @@
+name: aas229-workshop
+
+channels:
+  - http://ssb.stsci.edu/astroconda
+
+dependencies:
```

Comment:
Yes, I will mention `stginga` if I have time, since it is our add-ons to the "core" Ginga package. Glue 0.9 (as already listed in YAML) is fine. Whatever `stginga` that is in AstroConda is fine too.

Not so sure about `specviz`. @justincely told me he was going to do a new release but I have not heard much about it since.